packages/preview/flyingcircus/3.0.0/template/Main.typ

```diff
@@ -0,0 +1,130 @@
+#import "@preview/flyingcircus:3.0.0" : *
+
+#let title = "Sample Flying Circus Book"
+#let author = "Tetragramm"
+
+#show: FlyingCircus.with(
+  Title: title,
+  Author: author,
+  CoverImg: read("images/Cover.png", encoding: none),
+  Dedication: [It's Alive!!! MUAHAHAHA!],
+)
+
+#FCPlane(
+  read("Basic Biplane_stats.json"),
+  Nickname: "Bring home the bacon!",
+  Img: read("images/Bergziegel_image.png", encoding: none),
+)[
+This text is where the description of the plane goes. Formatting is pretty simple, but bold and italic don't work yet, I
+need to fake those.
+ 
+#underline[Words get underlined.]
+
+Leave an empty line or it will be the same paragraph
+ 
++ numbered list
+  + Sub-Lists!
++ Things! But you can have multiple lines for an item by indenting the next one, or just one long line.
+  - Sub-items!
+
+- Unnumbered list
+ 
+Break the column where you want it with `#colbreak()`
+
+Images can be added by doing `#image(path, width:blah, blah)`. The FC functions do fancy stuff though, so they need you
+to do `read(path,encoding:none)`
```

Comment:
I feel like a more idiomatic way would be to have the properties of `image` configured with a `set` rule, and take `image`s as parameters rather than bytes. For instance, this:

```typst
#let func = (img) => {
    image.decode(img, width: 80%)
}
```

Would become:

```typst
#let func = (img) => {
    set image(width: 80%)
    img
}
```